Jenkinsfile

```diff
@@ -0,0 +1,99 @@
+/**
+ * Cordova Jenkinsfile
+ */
+
+def platform = params?.PLATFORM?.trim()                      // e.g. "ios" or "android"
+BUILD_CONFIG = params?.BUILD_CONFIG?.trim()                 // e.g. "Debug" or "Release"
+CODE_SIGN_PROFILE_ID = params?.BUILD_CREDENTIAL_ID?.trim()   // e.g. "redhat-dist-dp"
+
+PROJECT_NAME = "cordova-showcase-template"
+CLEAN = true                          // Do a clean build and sign
+INFO_PLIST = "${PROJECT_NAME}/${PROJECT_NAME}-Info.plist"
```

Comment:
@wtrocki @mikenairn believes there may be an issue with the path here and the version of cordova. Does this look wrong ? Do we need to change it